media/pages/sk/faq/page.html

```diff
@@ -5,108 +5,199 @@ <h1>Často kladené otázky</h1>
 
 <h2 id="o-portali">O portáli</h2>
 
-<h3 id="co-je-chcemvediet-sk">Čo je chcemvediet.sk?</h3>
-<p>
-Chcemvediet.sk je portál, prostredníctvom ktorého môžete poslať žiadosť o informácie ktorejkoľvek
-inštitúcii, ktorá má povinnosť sprístupňovať informácie v zmysle zákona o slobodnom prístupe
-k informáciám č. 211/2000 Z.z. z jedného miesta, a ďalej spravovať svoje žiadosti o informácie.
-Zároveň môžete v databáze vyhľadávať medzi informáciami, ktoré už boli sprístupnené iným žiadateľom
-a sledovať štatistiky o vybavovaní žiadostí o informácie.
-</p>
+<div class="question">
+  <h3 id="co-je-chcemvediet-sk" class="chv-pointer" data-toggle="collapse"
+      data-target="#co-je-chcemvediet-sk-text">
+    <i class="faq-collapse-shown chv-icon chv-icon-lg icon-plus"></i><i class="hidden faq-collapse-hidden chv-icon chv-icon-lg icon-minus"></i><span class="question-heading-text">Čo je chcemvediet.sk?</span>
```

Comment:
@martin Medzi elementy bol _whitespace_, preto bolo potrebne to dat do jedneho riadku. Nejake riesenie [sem](https://stackoverflow.com/questions/2628050/ignore-whitespace-in-html) ktore sa ti paci?

Reply:
Suhlasim, jedno riesenie horsie ako druhe.

Na druhej strane, ked tu takto nechame dlhy riadok, tak casom zabudneme preco a "opravime" ho. Asi teda radsej pouzime nieco, z coho explicitne vyplyva, ze sa snazime odstranit whitespace. Pouzil by som asi komentare:

```
    <i class="faq-collapse-shown chv-icon chv-icon-lg icon-plus"></i><!--
 --><i class="hidden faq-collapse-hidden chv-icon chv-icon-lg icon-minus"></i><!--
 --><span class="question-heading-text">Čo je chcemvediet.sk?</span>
```

Reply:
@martinmacko47 Mozeme to nechat inline a aby sme predchadzali tomu, aby sme to v buducnosti odstranili, tak tam mozeme nechat comment 
```
<!-- Remove the space between inline-block elements -->
```